.github/workflows/pythonapp.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           doxygen
           cd ../python
           python -m sphinx -W -b html source build/html
+          rm -rf build/html/.doctrees
```

Comment:
My understanding is that the files that get actually uploaded are those generated with `doc/web/make_html.py`, right? If so, the addition of this line is probably unnecessary, the addition that matters is the one in `doc/web/make_html.py`.